app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>SampleApp</title>
```

Comment:
Research and apply "i18n" into project for translate string (using display on UI for end user)
https://guides.rubyonrails.org/i18n.html#providing-translations-for-internationalized-strings

Reply:
is it i need use YAML (.yml) or plain Ruby (.rb) files for storing my translations in SimpleStore?

Reply:
You should use .yml

Reply:
In my application.html.erb should be like this?
en:
  hello_world: Hello world!
  hello_flash: Hello flash
  success:
    'true':  'True!'
    'on':    'On!'
    'false': 'False!'
  failure:
    true:    'True!'
    off:     'Off!'
    false:   'False!'

pirate:
  hello_world: Ahoy World
  hello_flash: Ahoy Flash

I18n.t 'success.true'  # => 'True!'
I18n.t 'success.on'    # => 'On!'
I18n.t 'success.false' # => 'False!'
I18n.t 'failure.false' # => Translation Missing
I18n.t 'failure.off'   # => Translation Missing
I18n.t 'failure.true'  # => Translation Missing

Reply:
sorry, just found that i copy those coding to the wrong files